cursorless-talon/src/modifiers/containing_scope.py

```diff
@@ -58,6 +58,7 @@ def cursorless_containing_scope(m) -> str:
     "file": "document",
     "line": "line",
     "paint": "nonWhitespaceSequence",
+    "link": "url",
```

Comment:
This is just the first thing that came to mind. The name for this scope is totally up for debate

Reply:
Link is what comes to mind for me as well.

Reply:
Cool. To state the obvious `url` is a voice coding disaster haha